cuenca/resources/cards.py

```diff
@@ -35,6 +35,10 @@ class Card(Retrievable, Queryable, Creatable, Updateable):
     def last_4_digits(self):
```

Comment:
change to `last4` instead of `last_4_digits` to have consistency with https://github.com/samuelcolvin/pydantic/blob/5261fd05a0374b84ce2602d45990baf480fa2417/pydantic/types.py#L854

Reply:
That field is being used in Core and many other projects, I think it would be best if we created an issue and modified the field in the other projects, otherwise this change would create some PRs

Reply:
issue created:
https://github.com/cuenca-mx/cuenca-python/issues/148